### ISSUE TITLE
Meeting to discuss external software

### ISSUE THREAD

**Issue by Scarzy** [srobo/committee-tasks#6]

**Comment by Scarzy** [comment]
Will be called once new committees are in place, as it largely impacts them

**Comment by Scarzy** [comment]
In SR Calendar for 18/01/2023

**Comment by PeterJCLaw** [comment]
@Scarzy is that the right year?

**Comment by PeterJCLaw** [comment]
No, the year is wrong -- it's in a couple of weeks -- 2024-01-18. @Scarzy will announce in Slack/via email.

**Comment by Scarzy** [comment]
Meeting happened. Some matters were discussed. Main takeaway is that we need to be sure we fully understand what the underlying issues are that mean this was needed. Some data collection to occur and another meeting will happen, currently targeted for 22/02/24, but can be pushed to the other side of the competition if needed.

**Comment by Scarzy** [comment]
Minutes from last meeting; https://docs.google.com/document/d/1ADNq6AjXK5P-l9rChLe5Nj0FEchzRRPIMzEg4BLKqY4/edit#
Info to be gathered or task closed off

**Comment by PeterJCLaw** [comment]
@dianedowling has offered to raise this in the next trustees meeting and explore generically a risk register.
